air-quality-backend/tests/etl/in_situ/openaq_orchestrator_test.py

```diff
@@ -22,10 +25,11 @@
 )
 @patch("air_quality.etl.in_situ.openaq_orchestrator.fetch_in_situ_measurements")
 @patch("air_quality.etl.in_situ.openaq_orchestrator.fetch_forecast_data")
+@patch("os.environ.get")
```

Comment:
Minor, but you can do something like the following instead I think:

@mock.patch.dict(os.environ, {"STORE_GRIB_FILES": "True"})

Which is a bit easier to work out what's being mocked and means you don't need the param or the return_value setup
Assuming that makes sense there's a few examples in your test files but I won't comment on each.